### DISCUSSION TITLE
defaultValue is missing

### DISCUSSION THREAD

**Discussion by ivanafanasyeu** [react-hook-form/react-hook-form#2527] [General] [answered]
Why when I clean my input, this warning shows inside my console?
<img width="1172" alt="Screen Shot 2020-08-08 at 18 45 31" src="https://user-images.githubusercontent.com/25664253/89714432-578f5b00-d9a7-11ea-8785-7348fd8ce3ca.png">
<img width="1172" alt="Screen Shot 2020-08-08 at 18 45 41" src="https://user-images.githubusercontent.com/25664253/89714436-5d853c00-d9a7-11ea-8d2d-07b44e322306.png">
<img width="1172" alt="Screen Shot 2020-08-08 at 18 45 50" src="https://user-images.githubusercontent.com/25664253/89714440-62e28680-d9a7-11ea-8952-0c477eb19070.png">

Steps for reproduce in images:

```
<Controller
            defaultValue={false}
            control={control}
            name={name}
            render={({ onChange, value }) => (
                <MyInputComponentUsingNumberFormat
                    currency={currency}
                    name={name}
                    onChange={(v: number) => {
                        handleChange && handleChange(v);
                        onChange(v);
                    }}
                    prefix={prefix}
                    value={value}
                />
            )}
        />
```

**Comment by bluebill1049** [comment]
oh i see, it's a bug for dev tool message.

**Comment by bluebill1049** [comment, marked as answer]
do you have a codesandbox for this issue?

**Reply by ivanafanasyeu** [reply]
https://codesandbox.io/s/festive-noyce-wqutj

**Reply by ivanafanasyeu** [reply]
For reproduce, you have to type something inside input, and then make input clear

**Reply by bluebill1049** [reply]
please do this instead: `onChange(v.floatValue === undefined ? '' : va.floatValue);` default value shouldn't be `undefined` in Controller.

**Reply by ivanafanasyeu** [reply]
Thank you
